ql/src/test/results/clientpositive/llap/vector_between_in.q.out

```diff
@@ -269,16 +269,16 @@ STAGE PLANS:
             Map Operator Tree:
                 TableScan
                   alias: decimal_date_test
-                  filterExpr: (cdecimal1) IN (2365.8945945946, 881.0135135135, -3367.6517567568) (type: boolean)
+                  filterExpr: ((cdecimal1) IN (2365.8945945946, -3367.6517567568) or (cdecimal1) IN (881.0135135135)) (type: boolean)
```

Comment:
This looks like regression where expressions are not further simplified like before. May be create a follow up jira?

Reply:
Yes, you are right. This is a missed merging opportunity due to different type inferred for values (although they are all decimal, note that different precision,scale inferred for them). I have created https://issues.apache.org/jira/browse/HIVE-23222 to tackle this.